modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/utils/transformViewsItemProps.tsx

```diff
@@ -0,0 +1,180 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import {IView} from '@liferay/frontend-data-set-web';
+import React from 'react';
+
+import dateFormat from '../../../common/utils/dateFormat';
+import formatActionURL from '../../../common/utils/formatActionURL';
+
+import '../../../../css/main.scss';
```

Comment:
This feels a bit odd, because when we usually import `/css/main.css` at the top level, we expect this `main.scss` to be included in all modules, not just in certain cases. Maybe we could change it to `fileIcons.scss` / `typeIcons.scss`. I'm not entirely sure about the naming, but we should try to avoid using such a general `main.scss`.

Also, importing it in a `/utils` file is a bit unusual, it might make more sense in some components, but it’s okay, we can live with it.

Reply:
Changing it to ObjectDefinitionsIcons.scss and also move it to props_transformer/utils

Reply:
It's almos perfect, I think it better to move it to `/css/props_transformer` instead of `/css/props_transformer/utils`.

Reply:
ok, no problem, I will move it

Reply:
done a198c529a838a1c31e2e335e0dacaf0296469632